galaxy/wrapper/tool_dependencies.xml

```diff
@@ -1,9 +1,71 @@
 <?xml version='1.0' encoding='utf-8'?>
 <tool_dependency>
-    <package name="python" version="2.7.10">
-        <repository name="package_python_2_7_10" owner="iuc" />
+    <package name="numpy" version="1.9">
+        <repository name="package_python_2_7_numpy_1_9" owner="iuc" prior_installation_required="True" />
+    </package>
+    <package name="pybigwig" version="0.2.4">
+        <repository name="package_python_2_7_10_pybigwig_0_2_4" owner="iuc" prior_installation_required="True"/>
+    </package>
+    <package name="matplotlib" version="1.4">
+        <repository name="package_python_2_7_matplotlib_1_4" owner="iuc" prior_installation_required="True" />
+    </package>
+    <package name="scipy" version="0.14">
+        <repository name="package_python_2_7_scipy_0_14" owner="iuc" prior_installation_required="True" />
     </package>
-    <package name="deepTools" version="2.0">
-        <repository name="package_python_2_7_deeptools_2_0" owner="iuc"/>
+    <package name="pysam" version="0.8.3">
+        <repository name="package_python_2_7_pysam_0_8_3" owner="iuc" prior_installation_required="True" />
+    </package>
```

Comment:
Why is this change needed? I would prefer the old version? Can I revert this change?